### PR TITLE
feat: add the standalone advancer command

### PR DESCRIPTION
**Pull request by renan061**

### CODE REVIEW

**Review by torives — DISMISSED** [review]

File: cmd/cartesi-rollups-machines/main.go [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+)
+
+func main() {
+	err := Cmd.Execute()
+	if err != nil {
+		os.Exit(1)
+	}
+}
+
+func getDatabase(ctx context.Context, endpoint string, sslMode bool) (*repository.Database, error) {
```
Comment (fmoura) [inline cmd/cartesi-rollups-machines/main.go:42]: This could be moved to `startup` package, so it can be used by other commands

**Review by fmoura — DISMISSED** [review]

**Review by GMKrieger — DISMISSED** [review]

File: cmd/cartesi-rollups-machines/main.go [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+	"github.com/cartesi/rollups-node/internal/node/startup"
+	"github.com/cartesi/rollups-node/internal/repository"
+
+	"github.com/spf13/cobra"
+)
+
+const CMD_NAME = "advancer"
+
+var (
+	buildVersion = "devel"
```
Comment (marcelstanley) [inline cmd/cartesi-rollups-machines/main.go:26]: I'm assuming the Dockerfile will be updated to deal with production builds as we do at https://github.com/cartesi/rollups-node/blob/2206a7f1cd8937d8213954d2951cce3e561a2529/build/Dockerfile#L407

File: internal/node/advancer/advancer_test.go [hunk 3]
```diff
@@ -94,14 +97,18 @@ func (s *AdvancerSuite) TestRun() {
 		require.Len(repository.StoredResults, 3)
 	})
 
+	s.Run("Error/UpdateEpochs", func() {
+		s.T().Skip("TODO")
+	})
+
 	// NOTE: missing more test cases
```
Comment (marcelstanley) [inline internal/node/advancer/advancer_test.go:104]: This will be updated in a future PR, right?

**Review by vfusco — APPROVED** [review]
Merging as is. We will fix and improve it later.

**Review by mpolitzer — APPROVED** [review]

**Pull request merged**